### PR TITLE
[dc-subtitle] - minor update/refact

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/SubtitleFormats/DCSubtitle.cs [hunk 1]
```diff
@@ -512,10 +510,11 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                 System.Diagnostics.Debug.WriteLine(exception.Message);
             }
 
+            char[] splitChars = { ':', '.', ',' };
```
Comment (ivandrofly) [inline libse/SubtitleFormats/DCSubtitle.cs]: cache splitchars

File: libse/SubtitleFormats/DCSubtitle.cs [hunk 2]
```diff
@@ -63,20 +63,19 @@ public override bool IsMine(List<string> lines, string fileName)
                     xml.LoadXml(xmlAsString);
 
                     var subtitles = xml.DocumentElement.SelectNodes("//Subtitle");
-                    return subtitles != null && subtitles.Count > 0;
+                    return subtitles?.Count > 0;
                 }
                 catch (Exception ex)
                 {
                     System.Diagnostics.Debug.WriteLine(ex.Message);
-                    return false;
```
Comment (ivandrofly) [inline libse/SubtitleFormats/DCSubtitle.cs]: we already have one... line 73/74

File: libse/SubtitleFormats/DCSubtitle.cs [hunk 3]
```diff
@@ -63,20 +63,19 @@ public override bool IsMine(List<string> lines, string fileName)
...
                 }
             }
             return false;
         }
 
         public override string ToText(Subtitle subtitle, string title)
         {
-            string languageEnglishName;
+            string languageEnglishName = "English";
```
Comment (ivandrofly) [inline libse/SubtitleFormats/DCSubtitle.cs]: English is the default...

**Comment by niksedk** [comment]
If this is still relevant please add a new pr with info about why, what and unit tests

**Pull request closed without merging**